client/state/ui/guided-tours/test/contexts.js

```diff
@@ -14,7 +15,10 @@ import {
 	SOURCE_UNKNOWN,
 } from 'components/tinymce/plugins/wpcom-track-paste/sources';
 
+const WEEK_IN_MILLISECONDS = 7 * 1000 * 3600 * 24
```

Comment:
Duplicating this const in the test because including it like this:

`import { WEEK_IN_MILLISECONDS } from 'state/ui/guided-tours/contexts';`

...causes it to bail, presumably because of what else contexts.js imports.